### PR TITLE
Friendly block names

### PR DESCRIPTION
**Pull request by DMP9-MC**
This PR adds support for over 8.5k name combinations for blocks. The possible combinations are taken from the [drtshock essentials repository](https://github.com/drtshock/Essentials). Not much more to be said :)

### CODE REVIEW

File: prison-core/build.gradle [hunk 1]
```diff
@@ -22,6 +22,12 @@ apply plugin: 'idea'
 // Matches API level set in Prison.API_LEVEL
 version = 1.0
```
Comment (SirFaizdat) [inline prison-core/build.gradle]: Make this `1.1-SNAPSHOT`

**Comment by DMP9-MC** [comment]
Ready for merge?

**Comment by SirFaizdat** [comment]
No, you shouldn't remove JSON-Simple. It's used in some places.

**Comment by DMP9-MC** [comment]
Oh i thought it wasn't

**Comment by DMP9-MC** [comment]
Done

**Pull request merged**